Pages/page.js

```diff
@@ -0,0 +1,125 @@
+var EC = protractor.ExpectedConditions;
+
+class Page{
+    constructor(){
+        this.title="Page"
+    }
+
+    // GETTERS
+
+    get aboutUs(){
```

Comment:
Now when you have added Footer class maybe you can consider moving parts from here which are related for footer to Footer class (methods and getters) ?

Reply:
I removed footer.js, and moved methods to page.js